lib/ZitiEnroller.swift

```diff
@@ -22,6 +22,14 @@ import CZitiPrivate
 @objc public class ZitiEnroller : NSObject, ZitiUnretained {
     private static let log = ZitiLog(ZitiEnroller.self)
     private let log = ZitiEnroller.log
+    /// use the same loop for all enrollments, otherwise the logger's loop will be invalid after the first enrollment.
+    private static var loop: UnsafeMutablePointer<uv_loop_t> = {
```

Comment:
Looks like you expect this to get called exactly once. Do you test to make sure that's the case?

Reply:
Yes, this happens exactly once for the process, since it is "static".